modules/cloudfront-dashboard/README.md

```diff
@@ -0,0 +1,3 @@
+# Lightstep AWS CloudFront Dashboard
```

Comment:
This has been reviewed. And necessary changes have been made to pass github actions CI and local terraform apply.